README.md

```diff
@@ -1,7 +1,16 @@
 # hm-diag
 Helium Miner Diagnostics
 
-# Diagnostics JSON Layout
+**hm-diag** is a small website that displays diagnostic information about a hotspot.
+The website is only accessible if you are on the same network as the hotspot.
```

Comment:
Is access really restricted? I haven't dug through to find how we accomplish this.

Reply:
No access restriction, some people have forwarded port 80 to these and have it exposed on the internet -  not advised (https://www.shodan.io/search?query=title%3A%22Nebra+Hotspot+Diagnostics%22)

Reply:
Thanks, added a clarification: https://github.com/NebraLtd/hm-diag/pull/93/commits/a0fa55898a5c9c491c6dbb7d5b6624aa3f8e2cfa